homework.md

```diff
@@ -4,110 +4,112 @@
 
 What is 2 + 2?
 
-5
+Linus, 2+2 is 4, not 5, PUT ATTENTION STOP PLAYING POKEMON GO ON CLASSES!
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+White, Linus are you daltonic?
 
 ## Question 3
 
 What is the  2 * 3 * 6 * 1 * 10 * 34?
 
-1322
+12240, I need to accept I didn't even knew this.
 
 ## Question 4 
 
 What does HTML stand for?
 
-Hyper Tall Massive Language
+Hypertext Markup Language (HTML) (I didn't even knew this)
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+Making a website looks pretty, the hype for graphic designers and editors.
 
 ## Question 6
 
 Using CSS, how would I select every `<h1>` tag on the page?
 
 ```css
-#h1 {
+<h1> { Here is the section for heading elements
 
-}
+} </h1>
 ```
 
 ## Question 7
 
 What is JavaScript?
 
-A programming language used on the internet
+Javascript is a language that gives interface to the web pages adding interactive elements that catch the attention of the user. 
```

Comment:
I like how you describe it using your own words :)
Here's a question for you: can we use JS for creating things other than web pages? ;)